VM-RIght-Sizing/README.md

```diff
@@ -0,0 +1,28 @@
+# Dynatrace VM Right-Sizing Example
+
+This repository will help you deploy a configuration that will right-size your infrastructure using Dynatrace Automation Workflows. All configuration is deployed automatically using Dynatrace Configuration as Code solution Monaco.
+
+## Description 
+
+The example will help you detect improperly sized hosts, trigger an approval workflow in external tools such as Jira, ServiceNow ... and then react to that approval with a workflow to right-size the environment based on metadata stored in tags by calling the respective infrastructure automation APIs such as Azure, GCP, AWS, Kubernetes, OpenShift, VMWare, Ansible.
+
```

Comment:
Please add a few lines on what the example will actually do - how will it decide to right-size, what does it do with the slack/jira connections, etc.

Reply:
Added a detailed description of what each workflow will do. Please let me know if that's okay.

Reply:
That is great, thanks!